dashboard/app/models/user.rb

```diff
@@ -80,17 +80,20 @@
 
 class User < ActiveRecord::Base
   include SerializedProperties
-  # races: array of strings, the races that a student has selected, or nil if no selection was made
```

Comment:
Though `user.races` will be `nil`, we do not explicitly store `nil` in the DB. So I'm removing this comment.